src/v2/Apps/Artist/Routes/Consign/Components/Subheader.tsx

```diff
@@ -12,9 +12,9 @@ export const Subheader: React.FC<SubheaderProps> = ({
 }) => {
   return (
     <Box maxWidth="80%" m="auto">
-      <Serif element="h2" size="10" color={color}>
+      <Text as="h2" variant="title" color={color}>
         {children}
-      </Serif>
+      </Text>
```

Comment:
I presume this is unrelated. Is it just a best-practice/clean-up, or does it impact the page?

Reply:
I noticed there was some font weirdness looking at the page, and the old Serif component has been deprecated.